src/app/feature/map/search/search-bar/search-bar.component.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { AbstractControl, FormBuilder, FormGroup } from '@angular/forms';
+import { Observable } from 'rxjs';
+import { map, startWith } from 'rxjs/operators';
+import { GuideMapFeaturePoint } from '../../../../core/models';
+
+// TODO: Moved interfaces to another place
```

Comment:
Please move to interface folder, and rename to interface to map-search-bar-option